WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Quick Start/DashboardQuickStartCardCell.swift

```diff
@@ -20,8 +20,8 @@ final class DashboardQuickStartCardCell: UICollectionViewCell, Reusable, BlogDas
         return frameView
     }()
 
-    private lazy var tourStateView: QuickStarTourStateView = {
-        let view = QuickStarTourStateView()
+    private lazy var tourStateView: QuickStartTourStateView = {
```

Comment:
It took me quite a while to finally see the difference between this and before. 😆